dashboards/resources/queries/pod.libsonnet

```diff
@@ -0,0 +1,292 @@
+// queries path must match the path in the kubernetes-mixin template
+{
+  // CPU Queries
+  cpuUsageByContainer(config):: |||
+    sum by (k8s_container_name) (
+      max by (k8s_cluster_name, k8s_namespace_name, k8s_pod_name, k8s_container_name) (
+        rate(
+          k8s_pod_cpu_time_seconds_total{
+            k8s_cluster_name=~'${cluster}', k8s_namespace_name=~'${namespace}', k8s_pod_name=~'${pod}'
+          }[$__rate_interval]
+        )
+      )
+    )
+  |||,
+
+  cpuRequests(config):: |||
+    sum by (k8s_pod_name) (
+      max by(k8s_cluster_name, k8s_namespace_name, k8s_pod_name) (
+        k8s_container_cpu_request{
+          k8s_cluster_name=~'${cluster}', k8s_namespace_name=~'${namespace}', k8s_pod_name=~'${pod}'
+        }
+      )
+    )
+  |||,
+
+  cpuLimits(config):: |||
+    sum by (k8s_pod_name) (
+      max by(k8s_cluster_name, k8s_namespace_name, k8s_pod_name) (
+        k8s_container_cpu_limit{
+          k8s_cluster_name=~'${cluster}', k8s_namespace_name=~'${namespace}', k8s_pod_name=~'${pod}'
+        }
+      )
+    )
+  |||,
+
+  cpuThrottling(config):: '0',
+
+  // CPU Quota Table Queries
+  cpuRequestsByContainer(config):: |||
+    sum by (k8s_container_name) (
+      max by(k8s_cluster_name, k8s_namespace_name, k8s_pod_name, k8s_container_name) (
+        k8s_container_cpu_request{
+          k8s_cluster_name=~'${cluster}', k8s_namespace_name=~'${namespace}', k8s_pod_name=~'${pod}'
+        }
+      )
+    )
+  |||,
+
+  cpuUsageVsRequests(config):: |||
```

Comment:
I think there might be another metric which precalculates this for you (at least in the spec - check out https://opentelemetry.io/blog/2025/kubeletstats-receiver-metrics-deprecation/).

You could see if any of those metrics are present?